arquillian/api/pom.xml

```diff
@@ -96,6 +96,18 @@
       <artifactId>junit</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
```

Comment:
can we dependencyManage that version with a ${version.mockito}?  I think we might use it elsewhere also.